docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -197,15 +197,15 @@ The latest version of the `Authenticator` component has several differences from
 #### Slots
 
 All the slot locations have changed with the latest version of the `Authenticator`. To get a
-sense of the changes please check out the [Headers and Footers](../connected-components/authenticator/customization#headers--footers) section.
```

Comment:
Can you please explain why we need to change this?

Reply:
If we click it, or use cypress, the `href` would be `localhost:3000/connected-components/authenticator/customization#headers--footers`, but if we use `jsdom`, the href is just as the code `../connected-components/authenticator/customization#headers--footers`